static/app/views/dashboards/detail.spec.tsx

```diff
@@ -2185,6 +2193,12 @@ describe('Dashboards > Detail', function () {
           id: '1',
           title: 'Custom Errors',
         });
+        jest.mocked(useWidgetBuilderState).mockReturnValue({
```

Comment:
I mocked a bunch of these explicitly to avoid triggering the debounced stuff in the new state update provider. The updates here aren't actually that interesting, just as long as we know that adding the widget works with the correct state.